package.json

```diff
@@ -205,18 +205,23 @@
         {
           "id": "snyk.views.analysis.code.security",
           "name": "Code Security",
-          "when": "snyk:loggedIn && snyk:featuresSelected && snyk:codeEnabled && snyk:workspaceFound && !snyk:error"
+          "when": "snyk:loggedIn && snyk:featuresSelected && snyk:codeEnabled && !snyk:codeLocalEngineEnabled && snyk:workspaceFound && !snyk:error"
         },
         {
           "id": "snyk.views.analysis.code.quality",
           "name": "Code Quality",
-          "when": "snyk:loggedIn && snyk:featuresSelected && snyk:codeEnabled && snyk:workspaceFound && !snyk:error"
+          "when": "snyk:loggedIn && snyk:featuresSelected && snyk:codeEnabled && !snyk:codeLocalEngineEnabled && snyk:workspaceFound && !snyk:error"
         },
         {
           "id": "snyk.views.analysis.code.enablement",
           "name": "Code Security & Quality",
           "when": "snyk:loggedIn && snyk:featuresSelected && !snyk:codeEnabled && snyk:workspaceFound && !snyk:error"
         },
+        {
+          "id": "snyk.views.analysis.code.localEngine",
+          "name": "Code Security & Quality",
```

Comment:
Would it make sense to name it `Code Security & Quality [LOCAL]`?

Reply:
This would change the label on the tree to be different when user gets this message and may confuse them, raising the question - where's Code Security & Quality [NON-LOCAL] results then? 
Since Andy didn't talk about renaming it when we decided on it, I've kept it as-is 💄